src/finchlite/finch_assembly/cfg.py

```diff
@@ -0,0 +1,282 @@
+from .nodes import (
+    AssemblyNode,
+    Assign,
+    Block,
+    Break,
+    BufferLoop,
+    Call,
+    ForLoop,
+    Function,
+    GetAttr,
+    If,
+    IfElse,
+    Length,
+    Literal,
+    Load,
+    Module,
+    Repack,
+    Resize,
+    Return,
+    SetAttr,
+    Slot,
+    Stack,
+    Store,
+    TaggedVariable,
+    Unpack,
+    WhileLoop,
+)
+
+
+class BasicBlock:
```

Comment:
basic blocks, etc, should live in a separate folder from finch_assembly, because dataflow is supposed to apply to all 3 IRs. The CFGBuilder for FinchAssembly can stay in this file though.

Reply:
Treat the ControlFlowGraph like another IR. Put it in the dataflow directory, give it a cfg.py file that holds a PrinterContext.